PyInstaller/hooks/hook-enchant.py

```diff
@@ -18,3 +20,24 @@
     for d in files:
         for f in d[1]:
             datas.append((f, d[0]))
+elif is_darwin:
+    # Collecting dictionaries only works when pyenchant is installed via pip since the dictionaries installed via pip
+    # are stored next to the enchant library
+    binaries = []
+    files = exec_statement("""
+from enchant._enchant import e
+print(e._name)""").strip().split()
+    for f in files:  # Put the enchant library in lib/ so the enchant plugins can find it
+        binaries.append((os.path.join('lib', os.path.basename(f)), f))
```

Comment:
* Is pyenchant able to find the enchant.dylib when it is next the executable and not in 'lib'?
* Have you tested this hook with macports or homebrew?